src/converters/fromZigbee.ts

```diff
@@ -3173,6 +3173,10 @@ const converters1 = {
             if (hasAlreadyProcessedMessage(msg, model, msg.data.frameCounter, `${msg.device.ieeeAddr}_${commandID}`)) return;
             if (commandID === 224) return;
 
+            if (!msg.data.commandFrame.raw) {
```

Comment:
Instead of doing this, can we change:

```
const ID = `${commandID}_${msg.data.commandFrame?.raw.slice(0, 1).join('_')}`; // added a ? here
```

And add it to the lookup? 

```
'104_undefined': 'short_press_2_of_2'
```

Or alternatively, we change `if (commandID === 224) return;` to `if (commandID === 224 || commandID == 104) return;`, which is my preference as this only seems to be send once during pairing?

Reply:
If it's just this giving the error I propose to add it, it's much more user friendly.

Reply:
Add them all to the lookup you mean?

Reply:
This code applies to just the PTM 216Z, so not sure what actions are send.

I've updated the PR
- It will not give the error anymore, but log a `PTM 216Z: missing command` when there is no such command
- Added `short_press_2_of_2`

What do you think?

Reply:
Sounds good. This way users can report commands that log errors, and support can be added as needed.
Those 3 GP converters (215-216) cover lots of actual devices from what I've seen.